chef/cookbooks/crowbar/templates/default/crowbar-router-migration.sh.erb

```diff
@@ -35,17 +35,23 @@ set +x
 source /root/.openrc
 set -x
 
+if ssh $hostname "test -e $UPGRADEDIR/crowbar-router-migrated" ; then
+    log "l3 agent was already evacuated from host $hostname"
+    touch $UPGRADEDIR/crowbar-router-migration-ok
+    exit 0
+fi
```

Comment:
It feels somehow wrong for me to do this from within here. Have you considered doing it from the rails model in ```
evacuate_network_node``` (e.g. around here: https://github.com/crowbar/crowbar-core/blob/master/crowbar_framework/app/models/api/upgrade.rb#L776)